python/ray/serve/experimental/gradio_visualize_graph.py

```diff
@@ -39,6 +41,73 @@ def lazy_import_gradio():
     return _gradio
 
 
+def get_block(node, **kwargs):
+    gr = lazy_import_gradio()
+
+    return_type = node.get_return_type()
+    node_name = _DAGNodeNameGenerator().get_node_name(node)
+    if return_type is None:
+        logger.warning(
+            f"Return type for {node_name} node was not provided. "
+            "Defaulting to gr.Textbox."
+        )
+        return gr.Textbox(**kwargs)
+
+    if return_type == "int":
```

Comment:
Can you document where these strings come from? That helps future maintainers expand this list.